test/integration-test.py

```diff
@@ -540,28 +544,11 @@ def main():
     global exit_status
     exit_status = 0
 
-def run_chisel():
-    # TODO(https://github.com/letsencrypt/boulder/issues/2521): Add TLS-SNI test.
-
-    test_issuer()
-    test_expired_authz_purger()
-    test_ct_submission()
-    test_gsb_lookups()
-    test_multidomain()
-    test_expiration_mailer()
-    test_caa()
-    test_admin_revoker_cert()
-    test_admin_revoker_authz()
-    test_certificates_per_name()
-    test_ocsp()
-    test_single_ocsp()
-    test_dns_challenge()
-    test_renewal_exemption()
-    test_expired_authzs_404()
-    test_account_update()
-    test_stats()
-    if os.environ.get('BOULDER_CONFIG_DIR', '').startswith("test/config-next"):
-        test_sct_embedding()
+def run_chisel(test_case_filter):
+    print test_case_filter
+    for key, value in globals().items():
+      if callable(value) and key.startswith('test_') and re.search(test_case_filter, key):
```

Comment:
That's a fun bit of dynamic Python :snake: :running_man:

Reply:
✨very cool magic ✨